01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,186 @@ using InteractiveUtils
+He is very clear when commenting that the development of the jet engine had nothing to do with the discoveries of physicists researching, but thanks to the cleverness and antifragile heuristics based on trial and error that engineers have developed, although in academic books it is said that the process was the opposite. 
+Or in his main field, finances. It is unbelievable the way he describes how senior traders, that have been deploying several heuristics to make their trades, are much more complex than the ones generated by the pricing formulas that academics invented based on the trial-and-error experiences of people who are immersed in the market. 
+And this happened because, in the process of trying to generalizate this heuristics into formal equations, the academics are constantly introducing fragilities. 
+And the worst thing? After doing so, finance PhDs can't understand how traders can correctly assing prices to financial derivatives without knowing several theorems... 
+So the wheel is broken.
 
-So then, what role does Science play in all this and why is it useful for technology? As we already said, it is a tool that technology has to formalize all that chaotic based knowledge discovery. Formalization is very useful to teach and communicate some base knowledge and to build a solid foundation in order to keep expanding it. But it is also the way we found to draw out subjectivities regarding the understanding of our world.
+We are makers. We are Hackers. And, as makers that we are, we learn by doing. 
+That is another sign of how different technology is from the sciences. 
+Scientists don't learn science by doing it, but by doing labs and problem sets. 
+Scientists start out doing work that's perfect, in the sense that they're just trying to reproduce work someone else has already done for them. 
+Eventually, they get to the point where they can do original work. Whereas makers, from the start, are doing original work; it's just very bad. 
+So makers start original, and get good, and scientists start good, and get original.
```

Comment:
I don't quite follow the ideas in this paragraph. Please develop them a bit more

Reply:
Added example:
"That is another sign of how different technology is from the sciences. 
Scientists don't learn science by doing it, but by doing labs and problem sets. 
Scientists start out doing work that's perfect, in the sense that they're just trying to reproduce work someone else has already done for them.
For example when physics students go to the lab and perform the well-known pendulum experiment. Using the length of it and the period at which it pendulates, they are able to find the gravity acceleration using a formula that describes its motion: $T = 2π\sqrt[2]{\frac{L}{g}}$. They are not creating some new form of experiment, but recreating one that is well know throughout the scientific community and that has been performed thousands of times in different parts of the world. 
Eventually, they get to the point where they can do original work. Whereas makers, from the start, are doing original work; it's just very bad. "